### DISCUSSION TITLE
Crate+Hbase mode

### DISCUSSION THREAD

**Discussion by cmdares** [crate/crate#9805] [General]
The existing bigdata architecture adopts the classic HBase + elasticsearch. Due to the complexity of ES query syntax, I want to replace es with crate
to use SQL mode,  use the HBase + cratedb mode to build system.Is the idea realistic?
cratedb creates index for all fields. Except for the primary key, other fields are not stored.
The application layer obtains the primary key through the SQL query, and next to get the detailed data in HBase.
Advantage:
1、Open SQL capability
2、Reduce mass storage
3、Improve query and analysis performance

thanks!

**Comment by eazydripz** [comment]
Thank you.
